### ISSUE TITLE
Installer should use public start menu (and desktop) for links 

### ISSUE THREAD

**Issue by nico-dr** [corrados/jamulus#792]
if you are logged in as non administrator user (hopefull the most are) and you install Jamulus on a Win 10 PC, you will be asked for an administrator account and the installer will run under this (different) user with the result that the current logged in user will not find the usefull links to start Jamulus.

**Comment by ann0see** [comment]
Same issue here. (Thought of this a while ago). I was also a bit confused that it didn't show up in my start menu. Should be an easy fix.

**Comment by pljones** [comment]
The installer needs a good rewrite.

It should allow the individual user to decide how to install it.

I want to have an "all users" install.  Some people might not want that, though.

I don't want desktop shortcuts.  Some people might want them, though.

I only want the 64bit version installed.  And I want the default install location to be correct (not the 32bit location).

But some people may still need the 32bit version, as well, even when running on 64bit.

I want to specify what the install locations are.

Most of all, I want the installer to remember my choices.

**Comment by ann0see** [comment]
> The installer needs a good rewrite.

Probably yes. 

Nevertheless my draft pr would be a fast fix for that. 

I got really annoyed by this bug and if you don't find Jamulus on your desktop it's a very bad UX. Experienced users can always delete desktop icons, inexperienced users would be annoyed by the current behavior ("No, I can't see Jamulus!!! It's not installed!!!" "Yes it is! Go to Program Files>..." "How do I do that?"...)

**Comment by corrados** [comment]
> The installer needs a good rewrite.

Can someone try out the current implementation on https://github.com/corrados/jamulus/tree/improve_win_ci? I am pretty sure that this solves the issues. Unfortunately, I did not succeed in running it.

**Comment by ann0see** [comment]
I'm now trying it.

You need to set

` Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`

and there seem to be multiple other issues

**Comment by ann0see** [comment]
Now it does start compiling but throws an error after a few minutes. Nur sure, but there might be something wrong in the Jamulus Source code. 

We could port at least a few changes from this branch. Not 100% sure how to fix the power shell script yet, but it should be possible.

**Comment by ann0see** [comment]
Got it to work (run on external harddisk and admin and my patch).

![jamulusinstaller](https://user-images.githubusercontent.com/20726856/102988652-86503d00-4514-11eb-9460-402df1fe4867.png)



Unfortunately it installs jamulus into `C:\Program Files\Jamulus\F\jamulus\deploy\x86_64`...

**Comment by ann0see** [comment]
#794 could potentially fix this. @corrados could you please test it? You might have to change the path to VS 2015 

Here:

https://github.com/ann0see/jamulus/blob/1ffb86c4a2c5d168581d1a79b8f709946ad618f4/windows/deploy_windows.ps1#L97

And here:

https://github.com/ann0see/jamulus/blob/1ffb86c4a2c5d168581d1a79b8f709946ad618f4/windows/deploy_windows.ps1#L102

**Comment by corrados** [comment]
You seem to have changed the file in the meantime. I guess I have to change this:
```
    [string] $QtCompile32 = "msvc2015",
    [string] $QtCompile64 = "msvc2015_64",
```
I am a newbe regarding Windows powershell. When I try to call the script, I get:
```
.\deploy_windows.ps1 : Die Datei "C:\[...]\deploy_windows.ps1" kann nicht
geladen werden, da die Ausführung von Skripts auf diesem System deaktiviert ist. Weitere Informationen finden Sie
unter "about_Execution_Policies" (https:/go.microsoft.com/fwlink/?LinkID=135170).
In Zeile:1 Zeichen:1
+ .\deploy_windows.ps1
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : Sicherheitsfehler: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```
Do I have to run it as administrator?

**Comment by ann0see** [comment]
@corrados yes, you should run it as Administrator from a location near to the “root“ like C:\Jamulus

Before you can execute it, you also need to execute 

`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`

**Comment by corrados** [comment]
I have moved the jamulus directory to C:\ and run the power shell as admin. I still get an error:
```
C:\jamulus\windows\windows\ASIOSDK2
Move-Item : Ein Teil des Pfads konnte nicht gefunden werden.
In C:\jamulus\windows\deploy_windows.ps1:68 Zeichen:5
+     Move-Item -Path "$TempDir\$Name" -Destination "$WindowsPath\$Dest ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\corrad...mp\ASIOSDK2.3.2:DirectoryInfo) [Move-Item], DirectoryNotFoundException
    + FullyQualifiedErrorId : MoveDirectoryItemIOError,Microsoft.PowerShell.Commands.MoveItemCommand
```

**Comment by nico-dr** [comment]
You have to replace $PWD with C:\jamulus in your powershell script (or set this variable with your value)

**Comment by ann0see** [comment]
@corrados how do you run the Script? You need to run it from C:\jamulus by executing .\windows\deploy_windows.ps1 the error could also be related to something which is wrong with the temp path.

**Comment by nico-dr** [comment]
still some corrections in installer.nsi needed, the (public) desktop link points to c:\Jamulus.exe

**Comment by ann0see** [comment]
I thought I fixed this error?

**Comment by nico-dr** [comment]
You did, sorry my code wasn't up to date.

For QT v5.12.3 you have to set $QtCompile32 = "msvc2017" and $QtCompile64 = "msvc2017_64" and not 2019

I can confirm that bug #792 is fixed with the new installer.

The installer may need some more testing (and path imporvments) but now the installer is state of the art, good job!

**Comment by drummer1154** [comment]
Two small comments:

- Does the updated installer remove previous misplaced Jamulus x64 installations from C:\Program Files (x86)\Jamulus?
- What about compatibility to WinXP?

Cheers
-hg

**Comment by ann0see** [comment]
> Does the updated installer remove previous misplaced Jamulus x64 installations from C:\Program Files (x86)\Jamulus?

No. I don't think so. You should ask the person who originally wrote this script.


>  find better ways to get the directory locations in the ps file.

Maybe something with the $PSScriptRoot variable? https://stackoverflow.com/questions/5466329/whats-the-best-way-to-determine-the-location-of-the-current-powershell-script

**Comment by drummer1154** [comment]
> Does the updated installer remove previous misplaced Jamulus x64 installations from C:\Program Files (x86)\Jamulus?
> No. I don't think so. You should ask the person who originally wrote this script.

From my PoV this answer is not very cooperative. If the installer is updated, it still needs to be compatible with previous versions (where the related bug was documented at #738) and it shall also satisfy the above request https://github.com/corrados/jamulus/issues/792#issuecomment-749710366 - no matter who does the update.

Same with non-Win10 x86 systems.

**Comment by ann0see** [comment]
> If the installer is updated, it still needs to be compatible with previous versions

I totally agree. The problem is that I don't fully understand the script either (I didn't write it), there must be a way to fix this problem. We could detect the folder and remove it? But what about people who have both versions installed? Is this even possible? What do you think? 

> The installer may need some more testing

Agree. And I think we should also rewrite some parts (#785 , the comment by @pljones 
 which @drummer1154 mentioned, ...). That's still a lot of work to do.

**Comment by drummer1154** [comment]
@ann0see Thanks for this answer.

I think the way to detect what is installed is reading the registry.

> But what about people who have both versions installed?

I do not really understand this. What is meant by "both versions"? I do not currently see the possibility to have more than one version (x86 or x64) installed as the current NSIS script (wrongly) determines that Jamulus is always installed as x86 SW.

**Comment by ann0see** [comment]
Ok. I wasn't sure if it is possible to have 32Bit and 64Bit versions of Jamulus installed (and if there even is a 32Bit version).

> I think the way to detect what is installed is reading the registry.

Ok. So if a specific key is found, we remove the directory?

**Comment by ann0see** [comment]
Ok. For me `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Jamulus` shows Jamulus (remove only) as DisplayName. So can we therefore execute the value of UninstallString if it's found?

**Comment by drummer1154** [comment]
Sorry for slow response but formatting "Insert code" correctly took me some time ...
I can only refer to the current installer.nsi where $INSTDIR == "C:\Program Files (x86)\Jamulus\":
`; add reg keys so that software appears in Windows "Add/Remove Software"`
`WriteRegStr HKLM "${UNINST_KEY}" "UninstallString" '"$INSTDIR\${UNINSTALL_EXE}"'`
where
`!define UNINST_KEY        "Microsoft\Windows\CurrentVersion\Uninstall\${APP_NAME}"`
and
`!define APP_NAME          "Jamulus"`
would suggest that we could find the uninstaller via
`HKLM\Microsoft\Windows\CurrentVersion\Uninstall\Jamulus\UninstallString`
but - as already said - in Win10 it is really found in
`HKLM\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Jamulus\UninstallString`
as
`"C:\Program Files (x86)\Jamulus\Uninstall.exe"`.
So
`C:\>reg query HKLM\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Jamulus\ /v UninstallString`
would return
`HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Jamulus`
`UninstallString    REG_SZ    "C:\Program Files (x86)\Jamulus\Uninstall.exe"`
but I cannot say how to deduct this from the current installer.nsi - most likely Win10 automatically relocates the key to the correct location in the registry.
Therefore I **withdraw my proposal to read the registry**. Instead, the method to locate 
`"C:\Program Files (x86)\Jamulus\Uninstall.exe"`
should be using the original (x86) $INSTDIR\${UNINSTALL_EXE} (prior to setting the correct x64 path) or even simply checking if the fixed pathname "C:\Program Files (x86)\Jamulus\Uninstall.exe" exists and executing it.

WinXP?

**Comment by ann0see** [comment]
Ok. I already coded the "Registry" approach, but we can change it of course. If we have real 32 bit installs this could trigger the message all the time which would be really annoying. We should find something which only the old installer does.

See https://github.com/ann0see/jamulus/tree/win_installer for the old registry approach.

And I have no idea of Windows XP. My old XP machine doesn't really work so I can't check it.

**Comment by ann0see** [comment]
@drummer1154 Now I check for the Uninstall.exe file and run it automatically if the user selects yes. Sorry for all the goto statements.

**Comment by ann0see** [comment]
@pljones coming back to your request:


> The installer needs a good rewrite.
>
> It should allow the individual user to decide how to install it.
> 
> I want to have an "all users" install. Some people might not want that, though.

Todo

> I don't want desktop shortcuts. Some people might want them, though.

Todo (do we also need it for the start menu?)
On which page should we ask?
https://stackoverflow.com/questions/1517471/how-to-add-a-desktop-shortcut-option-on-finish-page-in-nsis-installer

> I only want the 64bit version installed. And I want the default install location to be correct (not the 32bit location).

Done

> But some people may still need the 32bit version, as well, even when running on 64bit.

https://github.com/corrados/jamulus/issues/792#issuecomment-751881268

That's tricky since the installer (as far as I understand) checks if you're running 32 Bit or 64 Bit Windows. That's not optimal.

> I want to override what the install locations are.

This should be possible now.

> Most of all, I want the installer to remember my choices.

Not sure, but we'd need a ini file for that probably.

**Comment by drummer1154** [comment]
> Is it ok to check if you're running 64Bit version of Windows but have Jamulus installed in the 32 Bit folder which we don't know of if it's 64 or 32 Bit?
> How do we check if a 64Bit version of Jamulus is in the wrong folder and not a 32Bit version which the user might have installed since he wants the 32Bit version of Jamulus?

As far as I understood the mechanism of the currently released installer (V3.6.2), it only checks whether the PC runs x64 or x86 Windows and subsequently installs either the x64 or x86 Jamulus version into the **wrong** path on x64 systems and into the **correct** path on x86 systems. So I think 

> I check for the Uninstall.exe file and run it automatically if the user selects yes

is sufficient. I do not want to copy my analysis documented in https://github.com/corrados/jamulus/issues/738#issuecomment-748506017.

Regarding 32bit Jamulus installation on 64bit systems: Is this really required? Maybe this needs to be answerded by @pljones. From my PoV there is no need for this. Keep it simple (and stupid) ...

Regarding WinXP support (as a 32bit only system): Maybe I was mistaken - I know that NSIS supports WinXP and therefore I can test Jamulus installation on WinxP - but the SW fails to run. So my question is whether Jamulus is supposed to run on WinXP or not - and if not, the installer shall refuse to install Jamulus on WinXP.

Two questions to @ann0see:
- Where can I find your NSIS Jamulus installer so I can test it? (NB: I do not have a Jamulus build environment.)
- How do you achieve stricken thru text here? This is an excellent way to mark mistakes in one's previous comments.

**Comment by ann0see** [comment]
> Where can I find your NSIS Jamulus installer so I can test it? (NB: I do not have a Jamulus build environment.)

I can upload it for you

> How do you achieve stricken thru text here? This is an excellent way to mark mistakes in one's previous comments.
Like this:
`~~wrong~~`: ~~wrong~~

**Comment by ann0see** [comment]
A bit off topic concerning ASIO:
* Should we add a statement saying "Please install an ASIO driver if you haven't already to the welcome page"? 
or even better:

* Should we detect if the user has an ASIO driver installed?

**Comment by pljones** [comment]
>     * Should we detect if the user has an ASIO driver installed?

This should be do-able - they're in the registry, I believe.  The ASIOSDK should have the path.

>     * Should we add a statement saying "Please install an ASIO driver if you haven't already to the welcome page"?

Perhaps even more strongly, given no driver has been detected: "It is strongly recommended you stop installation at this point and complete installation and testing of an ASIO driver before installing Jamulus."

(Perhaps it's a bit vicious... :) )

**Comment by ann0see** [comment]
It should be in 

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\ASIO`

The installer should show an additional page which tells the user that there's no ASIO Driver installed (maybe after the installation of the main software)? I don't really like the idea of stopping the installation since it's not a fatal and unfixable error.

**Comment by pljones** [comment]
> I don't really like the idea of stopping the installation since it's not a fatal and unfixable error.

Fixing and testing before installing Jamulus means the user doesn't have that to worry about as part of getting Jamulus running.

It should, at least, be made _very_ clear that Jamulus will fail to run successfully without an ASIO driver installed, so there's no real point continuing until one has been installed and tested.

**Comment by ann0see** [comment]
I would prefer to integrate a separate page only shown to the user if no ASIO driver was found. It should provide an easy way to install ASIO4ALL since users who see this page are probably inexperienced. 

It could look like this:

No ASIO driver found.

To provide the lowest possible audio latency, the Jamulus client needs an ASIO driver to run. We couldn’t detect an ASIO driver on your system. If you have an external audio interface, your manufacturer might provide an ASIO driver you should install now.

If you don’t have an external audio interface (this is very likely since you don’t have an ASIO driver installed), usually there’s no ASIO driver from the manufacturer and you need a generic driver like ASIO4ALL.

Do you want to download and install ASIO4ALL now?

[Yes] [No]

**Comment by corrados** [comment]
What is the status of the installer? I see the following open issues:

> Before you can execute it, you also need to execute Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser

This should be documented somewhere. Maybe in the header of the script and also in the Wiki (once the new installer is ready)?

> You have to replace $PWD with C:\jamulus in your powershell script (or set this variable with your value).

Is this already fixed? The script still does not run on my PC.

> You need to run it from C:\jamulus by executing something like .\windows\deploy_windows.ps1

This is a must to be fixed. I want to run the installer from my normal Jamulus development directory and I do not want to copy it to a special case. What cases the issue with the path? That should be solvable.

> Also the graphics must be updated,...

I see many TODOs above. What is the status?

> What about compatibility on WinXP? (NSIS supports it but Jamulus doesn't.)
> So my question is whether Jamulus is supposed to run on WinXP or not - and if not, the installer shall refuse to install Jamulus on WinXP.

The current version of Qt does not support WinXP. So Jamulus won't run on WinXP.

**Comment by ann0see** [comment]
> You have to replace $PWD with C:\jamulus in your powershell script (or set this variable with your value).

This should be fixed (the script now cd's into the jamulus folder).

> I want to run the installer from my normal Jamulus development directory and I do not want to copy it to a special case.

Me too. I don't exactly know what the problem is, but maybe it's related to directories with a space in their name

**Comment by ann0see** [comment]
In fact, now the $PWD bug should be fixed. 

> I want to run the installer from my normal Jamulus development directory and I do not want to copy it to a special case.

I think there's a problem in the installer.nsi file here: https://github.com/corrados/jamulus/blob/6e45a5efa333e48805157b4e5d9d4c787a22e7c9/windows/installer.nsi#L107

Something is wrong with the prefix variable (at least for me, it stops after a space in the path)

**Comment by ann0see** [comment]
@geheimerEichkater could you please have a look at the images here: https://github.com/corrados/jamulus/tree/improve_win_ci/windows ? We will have to update them.

@corrados Does the latest version work on your PC now? Do you have spaces in the path to the jamulus git repo?
> Before you can execute it, you also need to execute Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
> This should be documented somewhere. Maybe in the header of the script
Can you add it to the script? I will open a PR on the documentation repo.

> It should, at least, be made very clear that Jamulus will fail to run successfully without an ASIO driver installed, so there's no real point continuing until one has been installed and tested.

I'm not yet totally sure if we can (legally) download the ASIO4All driver automatically via the Jamulus installer. I think this is the best and most user friendly way. 

... which also hints at the next issue #785. How should we handle SmartScreen? Do we provide an online installer which automatically downloads the latest version of Jamulus?

**Comment by corrados** [comment]
> @corrados Does the latest version work on your PC now? Do you have spaces in the path to the jamulus git repo?

No white spaces. This is what I get using the latest code:

```
**********************************************************************
Using Qt binaries for Visual C++ located at
C:\Qt\5.12.3\msvc2015_64\bin
**********************************************************************

The Qt binaries for Microsoft Visual C++ 2017 (msvc2017) could not be located. Please install Qt with support for MSVC 2017 before running this
script,then call this script with the Qt install location, for example C:\Qt\5.12.3
In C:\Users\corrados\Documents\TMP\installerbranch\jamulus\windows\deploy_windows.ps1:163 Zeichen:9
+         Throw "The Qt binaries for Microsoft Visual C++ 2017 (msvc201 ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (The Qt binaries...le C:\Qt\5.12.3:String) [], RuntimeException
    + FullyQualifiedErrorId : The Qt binaries for Microsoft Visual C++ 2017 (msvc2017) could not be located. Please install Qt with support for MSVC 201
   7 before running this script,then call this script with the Qt install location, for example C:\Qt\5.12.3
```

**Comment by ann0see** [comment]
> Maybe it is easier to let Github do the installer generation.

I already tried that: https://github.com/ann0see/jamulus/runs/1640919706?check_suite_focus=true but getting QT to work with 32 and 64 bit support seems quite difficult.

How did you run this? What's the content of C:\Qt\5.12.3\msvc2015_64

**Comment by ann0see** [comment]
I have another idea: What happens if you remove https://github.com/corrados/jamulus/blob/85957edaf69b01a6fed2d07ac10c3a58e8e64f22/windows/deploy_windows.ps1#L122
until

https://github.com/corrados/jamulus/blob/85957edaf69b01a6fed2d07ac10c3a58e8e64f22/windows/deploy_windows.ps1#L138

and set the paths automatically?

**Comment by corrados** [comment]
I have changed the following code lines:
    [string] $QtInstallPath = "C:\Qt\5.15.1",
    [string] $QtCompile32 = "msvc2019",
    [string] $QtCompile64 = "msvc2019_64",
and now the script succeeds.

**Comment by ann0see** [comment]
Ok. So you use the 2019 build tools now?

**Comment by corrados** [comment]
I am using VS2019 since a while now, actually.

**Comment by ann0see** [comment]
Ah. Ok. If that's not a problem for you, I think we can leave it like that.

**Comment by ann0see** [comment]
@corrados I've assigned you to the wiki page and would like to have a review (if everything is ok there). Once we merge improve_win_ci into master, you can merge the mentioned pr on the documentation repo into changes.

**Comment by geheimerEichkater** [comment]
> could you please have a look at the images here: https://github.com/corrados/jamulus/tree/improve_win_ci/windows ? We will have to update them.

I see three bmp, those the ones you're addressing? @ann0see Could you pls provide screenshots for me how they look inside the install app on windows (I have no windows machine around)? I'm left a bit clueless as to how exactly they're being displayed and why two of them have a very unusual resolution/aspect ratio, maybe screenshots can clarify that. Also the icons (client, server) in the main branch had already been updated, will they have to be replaced here to avoid merging conflicts?

**Comment by ann0see** [comment]
Yes. The bmp files are loaded in the installer.

Uninstaller start page:
![StartUnInst](https://user-images.githubusercontent.com/20726856/103749146-671dd900-5005-11eb-9297-a7e91d610a9d.png)
One page in the installer/uninstaller:
![jamInst](https://user-images.githubusercontent.com/20726856/103749147-67b66f80-5005-11eb-9ef7-6986e3fb49c2.png)

Installer start page:
![StartInst](https://user-images.githubusercontent.com/20726856/103749149-67b66f80-5005-11eb-9c8b-b8385169d1bb.png)


> Also the icons (client, server) in the main branch had already been updated, will they have to be replaced here to avoid merging conflicts?

I've already replaced them. We could load them from src/res propably. 

We can also discuss this this evening?

**Comment by corrados** [comment]
I just tried out the new installer on a Windows 32 bit version and got a strange result. Previously, there was an old Jamulus version installed which I did not uninstall prior to installing the new version. Then the new installer seemed to work fine (no error). I let the "create desktop icon" and "run Jamulus" ticked and after clicking finish, the old Jamulus version was started. Even if I click on the newly created icon on the desktop, the old Jamulus version is started. When I look into the program files/jamulus directory, it seems only the old files are there, nothing new added.

**Comment by corrados** [comment]
I now uninstalled the old Jamulus version and started the new installer again. It seems it does not install anything in C:\Program Files.

**Comment by ann0see** [comment]
That's really strange. Although I couldn't test it on a 32 Bit PC. Did the installer tell you where the files are installed?

Your comment reminds me of another issue: 

We might need to remove old versions of Jamulus automatically with the installer.

**Comment by corrados** [comment]
> Did the installer tell you where the files are installed?

No, the installer did tell me nothing. The installation process was way too fast. It seems it left out the actual step of copying the files somehow.

**Comment by ann0see** [comment]
Ah, yes. I had this issue once too. Are the last lines of the ps1 script commented? If that’s the case, you might have to uncomment that.

**Comment by geheimerEichkater** [comment]
> We can also discuss this this evening?

Didn't get that far, though it sure was a pleasure getting to talk to you!

I´ve taken the screens (thanks a lot 👍) and updated them. Since the icon will already be on display in the upper left corner of the window, I would see it more as a branding opportunity rather than an analogy/display of what an un-/installation looks like and suggest the graphics this way:

![wizard-installer](https://user-images.githubusercontent.com/67019343/103885241-fd263200-50df-11eb-83d1-12b31e4e4cf9.png)

Unless there's a useablilty reason for having two different graphics, I'd simply suggest the use of the same splash for installing and uninstalling, would you (not) agree?

![wizard-eula](https://user-images.githubusercontent.com/67019343/103886987-a4a46400-50e2-11eb-9063-cf97ca6f39e4.png)

For the EULA I'd also suggest the extended graphic rather than the icon. 

If you're fine with that, I'll replace the bmp files in the branch.

**Comment by ann0see** [comment]
Yes, this looks great.

I don't think we necessarily need two different pictures for install/uninstall. Any other thoughts on this issue?

**Comment by ann0see** [comment]
Ok. Should be fixed now. I think the problem was that NSIS doesn't allow you to disable/enable sections. Not sure how clean my approach is, but it seems to work on 32 bit Windows 8.1 and 64 Bit Windows 10. The only problem I saw is that the installer now installs more than 100 MB (not sure why he now wants to install more files (I can't see more files in the program files directory???))

**Comment by corrados** [comment]
@ann0see What is the current status of the new Windows installer on the Git branch? Is it ready for merge?

**Comment by ann0see** [comment]
It is ready to merge once the graphics are done.

I will open a PR and explain there what still needs to be done.

**Comment by geheimerEichkater** [comment]
Graphics are updated, pull request is online.
